cocos/2d/CCLabel.cpp

```diff
@@ -1505,7 +1505,7 @@ void Label::onDrawShadow(GLProgram* glProgram, const Color4F& shadowColor)
     {
         Color3B oldColor = _realColor;
         GLubyte oldOPacity = _displayedOpacity;
-        _displayedOpacity = shadowColor.a * 255;
```

Comment:
Why do it like this? `shadowColor.a` is already a float

Reply:
I don't understand a question. shadowColor.a is a float, definitely.

Reply:
Sorry about it. I misunderstood it. Could you please provide a test case to reproduce it?

Reply:
Just create a Label using Label::createWithBMFont, enable shadow for it. Run action FadeOut for it.
You can see label fadeout, but shadow doesn't fade.